### ISSUE TITLE
No data retrived

### ISSUE THREAD

**Issue by ouranos6** [gieljnssns/kostalpiko-sensor-homeassistant#1]
Hello,

I manage to install and see the sensor in the interface but I have no information retrieved.

This is my configuration.yaml sensor section
```yaml
sensor:
- platform: kostal 
  host: 'http://192.168.1.36'
  username: pvwr
  password: pvserver
  monitored_conditions:
    - solar_generator_power
    - consumption_phase_1
    - consumption_phase_2
    - consumption_phase_3
    - current_power
    - total_energy
    - daily_energy
```

and as you can see below no datas... and believe me right now we have a lot of sun!
![image](https://user-images.githubusercontent.com/20718760/63004262-3e2dfe80-be7a-11e9-92b6-1d398e828aa2.png)

I'm using hassio version and I can't use PIP or at least I don't know how to do that...

I hope you can help me with that :) I'm super excited to integrate my little PV production to my Home Assistant interface.

**Comment by gieljnssns** [comment]
Do you have any errors in the log?
Try to remove the consumption and generator from your configuration.yaml. This should be for https://shop.kostal-solar-electric.com/en/piko-ba-sensor.html

**Comment by ouranos6** [comment]
nothing as i  can see.

you wrote:

```python
# REQUIREMENTS = [
#     'https://github.com/gieljnssns/KostalPikoPy/archive/'
#     'master.zip#pikopy==1.1.0']
```

Do I have to install something in hassio?

**Comment by ouranos6** [comment]
I found that I inverted the username and password, now I have this message when I restart

```
Error while setting up platform kostal
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 126, in _async_setup_platform
    SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/kostal/sensor.py", line 65, in setup_platform
    dev.append(PikoInverter(piko, sensor))
  File "/config/custom_components/kostal/sensor.py", line 81, in __init__
    self.update()
  File "/config/custom_components/kostal/sensor.py", line 114, in update
    self._state = self.piko.get_current_power()
  File "/usr/local/lib/python3.7/site-packages/kostalpyko/kostalpyko.py", line 78, in get_current_power
    return int(self._get_raw_content()[0])
IndexError: list index out of range
```

**Comment by gieljnssns** [comment]
> nothing as i can see.
> 
> you wrote:
> 
> ```python
> # REQUIREMENTS = [
> #     'https://github.com/gieljnssns/KostalPikoPy/archive/'
> #     'master.zip#pikopy==1.1.0']
> ```
> 
> Do I have to install something in hassio?

This is installed automatically.

**Comment by gieljnssns** [comment]
Do you have restarted ha another time?

I'm on vacation, I'll have a look when I get home.

**Comment by gieljnssns** [comment]
Are you still having troubles?
I've just released v2.2 in [HACS](https://github.com/hacs/integration)

**Comment by gieljnssns** [comment]
Closing this for now...
